builtin/builtin.go

```diff
@@ -12,9 +12,13 @@ var (
 )
 
 type Function struct {
-	Name     string
-	Func     func(args ...interface{}) (interface{}, error)
-	Opcode   int
+	Name    string
+	Func    func(args ...interface{}) (interface{}, error)
+	Opcode  int
+	Context bool
```

Comment:
Let's add a new field, instead of this `Context` field. 

    FuncWithContext func(ctx, args ...interface{}) (interface{}, error)